### DISCUSSION TITLE
Feature request: Allow workflow to be re-run using workflow file from different commit

### DISCUSSION THREAD

**Discussion by Ian1971** [community/community#36769] [Actions]
Currently if you are debugging a workflow, especially if it is dependent on pull requests etc, it is a real pain if, for example, as you have to make the yaml change, create a pull request, merge it, check the workflow, fix the bug repeat.

If you could re-run the failed one but choose (or upload) a specific yaml file to override the one in the commit that failed, it would save a lot of time and a lot of tiny commits.

**Comment by ossanna16** [comment]
Thank you for the feedback @Ian1971, I'm glad you found our community 🙂
